docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,9 @@
+
+# React Native WinRT Development
+
+The following documentation is intended for developers who wish to contribute to the React Native WinRT project.
```

Comment:
Should this be rolled into CONTRIBUTING.md? That's where I'd expect to see this kind of contributor-focused detail.

Reply:
I thought about putting it there initially, however the info in that file seems more geared towards CLA & code of conduct information and didn't seem to fit all that well. I've added a link to that file for now

Reply:
Fair enough. I've moved the bullet list there